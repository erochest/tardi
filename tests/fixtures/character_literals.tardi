'a' 'b' 'c'
'\n' '\t' '\r' '\'' '\\'
'ðŸ¦€' 'ä¸–' 'ç•Œ'
'\u41' '\u7A'
'\u{1F600}' '\u{1F4A9}'
