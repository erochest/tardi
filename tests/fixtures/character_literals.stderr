'a'
'b'
'c'
'\n'
'\t'
'\r'
'\''
'\\'
'ðŸ¦€'
'ä¸–'
'ç•Œ'
'A'
'z'
'ðŸ˜€'
'ðŸ’©'
